app/main/config.py

```diff
@@ -4,6 +4,7 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 class Config:
+  ENVIRONMENT = os.getenv("DEPLOY_ENV", "local")
   SECRET_KEY = os.getenv('SECRET_KEY', 'my_precious_secret_key')
```

Comment:
what is this secret key for?

Reply:
No idea and unrelated to this PR, but I am curious :joy: